lib/matplotlib/ticker.py

```diff
@@ -168,6 +169,15 @@
     long = int
 
 
+# Work around numpy/numpy#6127.
+def divmod(x, y):
```

Comment:
I am with @efiring, please add an underscore.  There are parts of mpl that pre-date the 'underscore means private' convention in python.